delete_column_materialized_view/deploy/0.0.1/cd-deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -e
+
+tb deploy 
+../utils/populate_with_backfill.sh "date" "analytics_sessions_mv_v1" "analytics_sessions_v1" "analytics_sessions_1" "analytics_sessions_mv" "" "timestamp"
```

Comment:
I deleted the double underscore and just kept a single one `_v1` to just work as a new resource but I keep getting an error in the backfill that I'm not sure how to solve.

Looking at the action log, it's detecting the change as renames. I'm not sure if it's the issue but is not what I was trying to do:

`renamed:	delete_column_materialized_view/datasources/analytics_sessions_mv.datasource -> delete_column_materialized_view/datasources/analytics_sessions_mv_v1.datasource`
`renamed:	delete_column_materialized_view/pipes/analytics_sessions.pipe -> delete_column_materialized_view/pipes/analytics_sessions_v1.pipe`

And then at line 182 I'm getting a:
`Pipe 'analytics_sessions_v1' not found`

Reply:
To debug the issue I went to the Environment related to this PR `tmp_ci__65` and I've seen the changes have not been deployed.

The reason is `tb deploy` does not deploy renamed resources (we have an issue about it). Rename is tricky because one thing is renaming a resource  that keeps the same content (schema or nodes) and a different thing is renaming a resource with a change which should be a new resource + drop the old one.

For now, if we don't support rename `tb deploy` should raise an error.

I'm going to see if I can undo the rename

Reply:
Good to know, and good to document it @alrocar

Reply:
After creating new resources instead of renaming, fixture tests started to fail, it's because we are using a pre-release version of the CLI and we need to hide the version warning.

It can be done adding `TB_VERSION_WARNING=0` as an env variable in the .tinyenv file, if you look at the [previous version](https://github.com/tinybirdco/use-case-examples/pull/65/files#diff-353024ca8238589bff7c0869eeb3f1b5c4570477d922a64c511d424920a14654) it had it, I guess `tb release generate --semver` preserves content of .tinyenv file so maybe it was modified by hand.

Reply:
Thank you a lot.